core/src/main/java/com/yahoo/oak/InternalOakBasics.java

```diff
@@ -166,7 +166,11 @@ V replace(K key, V value, OakTransformer<V> valueDeserializeTransformer) {
 
         for (int i = 0; i < MAX_RETRIES; i++) {
             BasicChunk<K, V> chunk = findChunk(key, ctx); // find orderedChunk matching key
-            chunk.lookUp(ctx, key);
+            try {
```

Comment:
The comment above speaks about orderedChunk, do we know it is not going to happen for hash?

Reply:
Can you please add an explanatory comment here?